### PR TITLE
fix memory prediction calculator

### PR DESCRIPTION
**Pull request by ivanmorozov333**
### Changelog entry 

fix memory prediction

### Changelog category 

* Bugfix 

### Additional information

...

### CODE REVIEW

File: ydb/core/tx/columnshard/engines/changes/general_compaction.cpp [hunk 1]
```diff
@@ -275,4 +275,30 @@ std::shared_ptr<TGeneralCompactColumnEngineChanges::IMemoryPredictor> TGeneralCo
...
 
+ui64 TGeneralCompactColumnEngineChanges::TMemoryPredictorChunkedPolicy::AddPortion(const TPortionInfo& portionInfo) {
+    SumMemoryFix += portionInfo.GetRecordsCount() * (2 * sizeof(ui64) + sizeof(ui32) + sizeof(ui16));
+    ++PortionsCount;
+    THashMap<ui32, ui64> maxChunkSizeByColumn;
+    for (auto&& i : portionInfo.GetRecords()) {
+        SumMemoryFix += i.BlobRange.Size;
+        auto it = maxChunkSizeByColumn.find(i.GetColumnId());
+        if (it == maxChunkSizeByColumn.end()) {
+            it = maxChunkSizeByColumn.emplace(i.GetColumnId(), i.GetMeta().GetRawBytesVerified()).first;
```
Comment (k-morozov) [inline ydb/core/tx/columnshard/engines/changes/general_compaction.cpp:286]: why do we assign a new value to the it variable? we will overwrite it at the next iteration.

**Review by nsofya — DISMISSED** [review]

File: ydb/core/tx/columnshard/engines/changes/general_compaction.h [hunk 2]
```diff
@@ -50,27 +50,13 @@ class TGeneralCompactColumnEngineChanges: public TCompactColumnEngineChanges {
 
     class TMemoryPredictorChunkedPolicy: public IMemoryPredictor {
     private:
-        ui64 SumMemory = 0;
+        ui64 SumMemoryDelta = 0;
+        ui64 SumMemoryFix = 0;
         ui32 PortionsCount = 0;
         THashMap<ui32, ui64> MaxMemoryByColumnChunk;
+        std::map<ui64, std::vector<ui32>> RatedColumnIds;
```
Comment (k-morozov) [inline ydb/core/tx/columnshard/engines/changes/general_compaction.h:57]: I didn't find using RatedColumnIds. why did you add it?

**Review by k-morozov — APPROVED** [review]
